OpenRA.Mods.Common/Traits/World/PathFinder.cs

```diff
@@ -110,10 +110,13 @@ public List<CPos> FindUnitPathToRange(CPos source, SubCell srcSub, WPos target,
 			// Correct for SubCell offset
 			target -= world.Map.Grid.OffsetOfSubCell(srcSub);
 
+			var rangeLengthSquared = range.LengthSquared;
+			var map = world.Map;
+
 			// Select only the tiles that are within range from the requested SubCell
 			// This assumes that the SubCell does not change during the path traversal
-			var tilesInRange = world.Map.FindTilesInCircle(targetCell, range.Length / 1024 + 1)
-				.Where(t => (world.Map.CenterOfCell(t) - target).LengthSquared <= range.LengthSquared
+			var tilesInRange = map.FindTilesInCircle(targetCell, range.Length / 1024 + 1)
+				.Where(t => (map.CenterOfCell(t) - target).LengthSquared <= rangeLengthSquared
 							&& mobile.Info.CanEnterCell(self.World, self, t));
```

Comment:
Likely insignificant, but since we're making changes here anyway, might as well use the cached `world` instead of `self.World` here?

Reply:
Checked.